tests/unit_tests/test_metagenome.py

```diff
@@ -1,3 +1,4 @@
+import pandas as pd
```

Comment:
Need to include copyright notice and explanation of the script at the top.

Reply:
Do you think this (what the script does) is necessary for all test scripts? These are somewhat self-explanatory -- i.e. this tests the `metagenome.py` file

Reply:
Yes, but if the codebase becomes a lot bigger, it might not be immediately obvious what directory the tested script is in.